CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+Release Notes
+=============
+
+Version 0.0.3
+-------------
+
+Released PENDING
+
+* Added `py.test` fixtures for managing session in tests.
```

Comment:
nitpick: i think the project calls itself "pytest"

(or are you part of [savethedot](https://twitter.com/search?q=savethedot)? :)